packages/api/src/AssignmentFile/AssignmentFile.resolver.ts

```diff
@@ -0,0 +1,47 @@
+import { UseGuards } from '@nestjs/common';
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
+
+import { GQLAuthGuard } from '../Auth/GQLAuth.guard';
+import { AssignmentFile, CreateAssignmentFileInput, UpdateAssignmentFileInput } from './AssignmentFile.entity';
+import { AssignmentFileService } from './AssignmentFile.service';
+
+@Resolver('AssignmentFile')
+export class AssignmentFileResolver {
+  constructor(private readonly assignmentFileService: AssignmentFileService) {}
+
+  @UseGuards(GQLAuthGuard)
+  @Query(() => [AssignmentFile])
+  assignmentFiles() {
+    return this.assignmentFileService.findAll();
+  }
```

Comment:
Remove this. I don't think it will ever be used like that. We will allmost always want the file in reference to an individual assignment